tests/test-cases/Group1-Docker-Commands/1-17-Docker-Network-Connect.robot

```diff
@@ -19,12 +19,47 @@ Suite Setup  Install VIC Appliance To Test Server  certs=${false}
 Suite Teardown  Cleanup VIC Appliance On Test Server
 
 *** Test Cases ***
-Connect container to a new network
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} network create test-network
+Connect containers to multiple networks overlapping
```

Comment:
'Connect containers to multiple bridge networks' is a slightly more descriptive name.

Note that when a container is connected to an external vSphere network and a bridge network, a unique interface for each network is set up.